src/views/Login.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="body">
+    <b-card style="width: 600px;" class="cardsita">
+      <b-card-header header-bg-variant="white">
+        <h3>Login</h3>
+      </b-card-header>
+        <b-container>
+          <form>
+            <b-row class="pt-3">
+              <label for="email">Email:</label>
```

Comment:
the for="" attribute is not linking the label to the field since you did not give the input the "email" id.

Reply:
same for password

Reply:
Done